app/models/type.rb

```diff
@@ -0,0 +1,6 @@
+class Type < ApplicationRecord
+  has_one_attached :image
+  belongs_to :user
+  has_many :attributes_association, class_name: 'Attribute', dependent: :delete_all
```

Comment:
@basti when i put has_many :attributes, a have error: "You tried to define an association named attributes on the model Type, but this will conflict with a method attributes already defined by Active Record. Please choose a different association name.", so that is why i changed name to :attributes_association.